packages/lexicon/src/validators/complex.ts

```diff
@@ -123,7 +123,7 @@ export function object(
   // required
   if (Array.isArray(def.required)) {
     for (const key of def.required) {
-      if (!(key in value)) {
+      if (typeof value[key] === 'undefined') {
```

Comment:
one of my least fav JS quirks 😒 
"no it's not undefined, it's defined as undefined"